runtime/near-vm-logic/src/tests/alt_bn128.rs

```diff
@@ -109,12 +109,31 @@ fn test_alt_bn128_g1_multiexp() {
     }
 
     check_ok(&le_bytes![], &le_bytes![0x0 0x0]);
+    check_ok(
+        &le_bytes![
+            0x2d6b17489d86fcd5f91e8e92eb55081d8cb4413e408047249ef4fb5baa1b518b 0x1e4d0a30dbadd9dad40f7847c7013754ded8d0371c052d19f01453f4ae1506d7 0x1,
```

Comment:
I’m confused by formatting in this file.  Sometimes the data is on separate line, sometimes the whole thing is a single line.  Commas also seem to be used arbitrarily.  Furthermore, I’d wrap all the buffers at the space.  It’s probably too much noise to change it all though so whatever.